### ISSUE TITLE
vit vat work incorrect with <br> and  <br />.   

### ISSUE THREAD

**Issue by gerardroche** [NeoVintageous/NeoVintageous#161]
<a href="https://github.com/starcwl"><img src="https://avatars0.githubusercontent.com/u/12868797?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [starcwl](https://github.com/starcwl)**
_Friday Mar 18, 2016 at 10:03 GMT_
_Originally opened as https://github.com/guillermooo/Vintageous/issues/1062_

----

If I input "vit" or "vat" after rel, It select wrong tag.

```
<head>
    _<link rel= "shortcut icon" href="favicon.png">
</head>_
```

**Comment by gerardroche** [comment]
<a href="https://github.com/starcwl"><img src="https://avatars0.githubusercontent.com/u/12868797?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [starcwl](https://github.com/starcwl)**
_Friday Mar 18, 2016 at 10:06 GMT_

----

```
<head>
    _<link rel= "shortcut icon" href="favicon.png">
</head>_
```

if I input "vit" or "vat" after rel, It select wrong tag.

**Comment by gerardroche** [comment]
@starcwl This has been fixed in NeoVintageous 1.6.0 (NeoVintageous is a fork of the discontinued Vintageous plugin).
